### DISCUSSION TITLE
I am trying to deploy simple flask code on vercel  ,Error comes This Serverless Function has crashed.

### DISCUSSION THREAD

**Discussion by Rashi2608** [vercel/community#7330] [Help]
### Summary

![image](https://github.com/vercel/community/assets/138390220/9121470f-7e9e-4c91-bdd0-c5e7777700d5)

![image](https://github.com/vercel/community/assets/138390220/2c1de916-d414-408b-8c34-6f7ff4fb0e9f)
![image](https://github.com/vercel/community/assets/138390220/7a4fb5af-13b2-46ee-907f-c4634e591bf3)
![image](https://github.com/vercel/community/assets/138390220/369b1cf0-45f5-4d51-8914-f5da86fe8563)

can some help me please i am trying since yesterday

### Example

https://github.com/Rashi2608/Nakshtra.git

### Steps to Reproduce

https://github.com/Rashi2608/Nakshtra.git    just import this and try to deploy

**Comment by pawlean** [comment]
Hi @Rashi2608!

I'd love to help here. Can you re-send your GitHub repo? The link you provided didn't work.

[a comment by an automated account is not shown]
